jobs/involuntary-dissolutions/tests/unit/test_involuntary_dissolutions.py

```diff
@@ -217,3 +278,44 @@ def test_stage_2_process_update_business(app, session, test_name, status, step):
         assert batch_processing.trigger_date.date() == datetime.utcnow().date() + datedelta(days=30)
     else:
         assert batch_processing.trigger_date == TRIGGER_DATE
+
+@pytest.mark.parametrize(
+    'test_name, status, step', [
+        (
+            'DISSOLVE_BUSINESS',
+            BatchProcessing.BatchProcessingStatus.COMPLETED,
+            BatchProcessing.BatchProcessingStep.DISSOLUTION
+        ),
+        (
+            'MOVE_BACK_TO_GOOD_STANDING',
+            BatchProcessing.BatchProcessingStatus.WITHDRAWN,
+            BatchProcessing.BatchProcessingStep.WARNING_LEVEL_2
+        ),
+    ]
+)
+@pytest.mark.asyncio
+async def test_stage_3_process(app, session, test_name, status, step):
+    """Assert that businesses are processed correctly."""
+    business = factory_business(identifier='BC1234567')
+    batch = factory_batch(status=Batch.BatchStatus.PROCESSING)
+    batch_processing = factory_batch_processing(
+        batch_id=batch.id,
+        business_id=business.id,
+        identifier=business.identifier,
+        step=BatchProcessing.BatchProcessingStep.WARNING_LEVEL_2,
+        created_date=CREATED_DATE,
+        trigger_date=TRIGGER_DATE
+    )
+
+    if test_name == 'MOVE_BACK_TO_GOOD_STANDING':
+        business.last_ar_date = datetime.utcnow()
+        business.save()
+
+    with patch('involuntary_dissolutions.put_filing_on_queue'):
+        qsm = MagicMock()
+        await stage_3_process(app, qsm)
```

Comment:
can we have an assertion check that verifies that `put_on_filing_queue` is called for the `DISSOLVE_BUSINESS` test?